words/islam.yaml

```diff
@@ -14,7 +14,7 @@ versions:
       english: Islam
       french: ''
       hebrew: ''
-      portuguese: ''
+      portuguese: Islão/Islã
```

Comment:
When you have more than one translation it needs to be written in a slightly different way. Like this: I'll update this change.

```
       portuguese:
         - Islão
         - Islã
```